clients/admin-ui/src/features/organization/OrganizationForm.tsx

```diff
@@ -0,0 +1,152 @@
+import { Box, Button, Stack, useToast } from "@fidesui/react";
+import { SerializedError } from "@reduxjs/toolkit";
+import { FetchBaseQueryError } from "@reduxjs/toolkit/dist/query";
+import { Form, Formik, FormikHelpers } from "formik";
+import { useMemo } from "react";
+import * as Yup from "yup";
+
+import { CustomTextInput } from "~/features/common/form/inputs";
+import { getErrorMessage, isErrorResult } from "~/features/common/helpers";
+import { errorToastParams, successToastParams } from "~/features/common/toast";
+import {
+  DEFAULT_ORGANIZATION_FIDES_KEY,
+  useUpdateOrganizationMutation,
+} from "~/features/organization";
+import { Organization } from "~/types/api";
+
+// NOTE: A valid Organization is _required_ to use this form - it doesn't
+// support creating/deleting Organizations. Since Fides will automatically
+// create the "default_organization" on startup, this Organization should be
+// fetched prior to rendering
+interface OrganizationFormProps {
+  organization?: Organization;
```

Comment:
I ended up doing something subtle here which I thought was nice - if the `organization` prop is null, it renders the form in a "loading" state. That means it can fully render immediately but then the data populates the fields when it's ready.

I can update the comment though, that comment was from _before_ I tweaked that!

Reply:
Updated the comment